.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+    "extends": ["airbnb", "prettier"],
+    "plugins": ["prettier"],
+    "parser": "babel-eslint",
+    "rules": {
+        "prettier/prettier": ["error"],
+        "react/jsx-filename-extension": 0,
+        "experimentalDecorators": 0,
+        "react/jsx-indent": [2, 4]
+    },
+    "globals": {
+        "describe": false,
+        "it": false,
+        "expect": false,
+        "shallow": false
+    }
+}
```

Comment:
Might makes sense to remove `.json` from the file name, so we can have consistency with other config files

Reply:
sorry. that is deprecated
see: https://eslint.org/docs/user-guide/configuring#configuration-file-formats

Reply:
Oh ok!